frontend/public/style/_vars.scss

```diff
@@ -19,26 +19,13 @@ $fa-font-path: $consoleWebDistPath + '/';
 // Use PatternFly 4 font family in PatternFly 3.
 $font-family-base: var(--pf-global--FontFamily--sans-serif);
 $font-size-base: 14px;
```

Comment:
This is no longer needed since the Bootstrap 3 default is 14px and we no longer include the PatternFly override.

Reply:
removed